slam_toolbox:
  ros__parameters:
    mode: "mapping"  # The mode you want to use: "mapping", "localization", etc.
    resolution: 0.05  # Resolution of the map (in meters per cell)
    map_update_interval: 1.0  # Time interval to update the map (in seconds)
    map_width: 1 # 1024 # Width of the map in cells
    map_height: 1 # 1024 # Height of the map in cells
    origin_x: 0.0  # X coordinate of map origin (in meters)
    origin_y: 0.0  # Y coordinate of map origin (in meters)
    origin_z: 0.0  # Z coordinate of map origin (in meters)
    initial_pose: [0.0, 0.0, 0.0]  # Initial pose of the robot (x, y, theta)
    transform_publish_period: 0.05  # How often to publish the transform
    odom_frame: "odom"  # The name of the odometry frame
    base_frame: "base_link"  # The name of the base frame
    laser_scan_topic: "/robot2/scan"  # The topic for your laser scan (adjust if needed)
    map_frame: "map"  # The name of the map frame
    scan_topic: "/robot2/scan"  # Laser scan topic for mapping
    use_odometry: true
    use_sim_time: false
